### DISCUSSION TITLE
Staking

### DISCUSSION THREAD

**Discussion by TiffaC** [iotaledger/firefly#8629] [Help & Support]
HELP: Possibly 2 years ago I staked my Iota assets to obtain Shimmer and I believe something else, which I received over a period of time. Now I have no record of that staking or of the tokens received in my account. How, and where, do I find any records of these transactions and where are the tokens I received?

**Comment by begonaalvarezd** [comment]
Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version (https://firefly.iota.org/) in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

You can join our discord if you have more questions https://discord.iota.org/
